src/win/core.c

```diff
@@ -159,6 +160,17 @@ static void uv_init(void) {
 }
 
 
+/* The number of milliseconds in one second. */
+#define UV__MILLISEC 1000
```

Comment:
`UV__MILLISECONDS_PER_SECOND`?  Just `UV__SECOND` doesn't tell you the granularity.